src/main/java/com/exasol/releaserobot/validation/PlatformValidatorFactory.java

```diff
@@ -29,10 +30,11 @@ public static PlatformValidator createPlatformValidator(final GitBranchContent r
         final PlatformName releasePlatform = platform.getPlatformName();
         if (releasePlatform == GITHUB) {
             return new GitHubPlatformValidator(repositoryContent, (GitHubPlatform) platform, validationReport);
+        } else if (releasePlatform == MAVEN) {
+            return new MavenPlatformValidator(repositoryContent, validationReport);
```

Comment:
passing this validation report as a state might bring undesired side effects. We should consider to return it when calling the public API